src/components/Upload.vue

```diff
@@ -1,8 +1,8 @@
 <template lang="pug">
 v-card.upload-wrapper(flat)
-  v-card-title(primary-title)
-    div
-      slot(name="toolbar")
+  slot(name="header")
+    v-card-title(primary-title)
+      div
```

Comment:
Is this `div` necessary?

Reply:
Apparently so. `v-card-title` is set up as a horizontal flexbox (why? not sure) so direct child elements will be laid out horizontally. All Vuetify [examples](https://vuetifyjs.com/en/components/cards) that use `v-card-title` put all content within a single `div`.

Reply:
Even a core developer states that it is a flexbox for "reasons unknown" https://github.com/vuetifyjs/vuetify/issues/3558